databind/src/main/java/tech/ydb/yoj/databind/expression/ModelField.java

```diff
@@ -67,25 +76,25 @@ public Stream<ModelField> flatten() {
 
     @NonNull
     public FieldValue validateValue(@NonNull FieldValue value) {
-        if (value.isTuple()) {
-            value.getTuple().streamComponents()
+        if (value instanceof TupleFieldValue tupleValue) {
```

Comment:
logic under instanceof for interface looks little bit strange. Maybe it possible to make part of interface?
```
if(!value.check(fieldValueType)) {
    if (userFieldPath != null) {
         throw value.buildUserException(userFieldPath)
    } else {
        throw new IllegalArgumentException("formatted message")
    }
}

```